src/platform/forms-system/src/js/components/ContactInfoReview.jsx

```diff
@@ -30,13 +32,12 @@ const ContactInfoReview = ({ data, editPage, content, keys }) => {
 
   useEffect(
     () => {
-      if (
-        window.sessionStorage.getItem(CONTACT_EDIT) === 'true' &&
-        editRef?.current
-      ) {
+      if (getReturnState() === 'true,' && editRef?.current) {
         // focus on edit button _after_ editing and returning
-        window.sessionStorage.removeItem(CONTACT_EDIT);
-        setTimeout(() => focusElement(editRef.current));
+        clearReturnState();
+        setTimeout(() =>
+          focusElement('button', {}, editRef.current.shadowRoot),
+        );
```

Comment:
Is this timeout just to push this focusElement to the end of the event loop/task queue?

Reply:
> I think the removal of the reliance of the session storage value is a good call.
It didn't really get removed 😞 - the code was switched to use functions that manipulate session storage